app/bundles/InstallBundle/Configurator/Step/CheckStep.php

```diff
@@ -246,4 +251,27 @@ public function update(StepInterface $data): array
 
         return $parameters;
     }
+
+  private function getParameterLoader(): ParameterLoader
```

Comment:
Any reason why we do not use CoreParametersHelper here?

Reply:
Thank you, adjusted this.